### DISCUSSION TITLE
I cannot find realmlist.wtf under wow client folder

### DISCUSSION THREAD

**Discussion by njuwelkin** [azerothcore/azerothcore-wotlk#13920] [Guides & Tips]
as title.
and it doesn't work if i create one.

**Comment by virgo77** [comment]
I hope you are not trying to play with AzerothCore through the WotLK Classic BNet Client. It will not work... You need an old client version, as it must be the latest build in 2010! So, client 3.3.5a build 12340.
you can download different clients from this website: https://www.wowdl.net/fichiers/clients/

The realmlist.wtf file is under the "data\enUS" folder or "data\enGB" folder, or whatever your language is.

The original realmlist.wtf file is like this one:
```
set realmlist us.logon.worldofwarcraft.com
set patchlist us.version.worldofwarcraft.com
set realmlistbn ""
set portal us
```

What you have to do, if the file is missing, is to create a new one under one of the folders I gave above, and just add a simple line which contains the IP Address of your server, like this: `set realmlist 192.168.1.4`

That's all

Have a good game

**Comment by njuwelkin** [comment]
thank you so much. yes, i used the wrong client.
however I encountered another problem. wowdl.net always return 1020/access denied.
Anyway it's not a block problem. I'll try to find it from other place later. Thanks again!

**Comment by virgo77** [comment]
wowdl.net was under maintenance yesterday. Maybe the site is still not working 100% perfect.

you can try this one too: https://www.cosmicspacestation.gg/wotlk-3-3-5-client-download-wrath-of-the-lich-king-client/

And, if you look a bit on the link above, you can find some modified clients that includes Legion Graphics. It is fan works to make the game more beautiful. You will have better characters and players graphics, etc.:
- Here: http://forum.warmane.com/showthread.php?t=433477&s=20fd269d55aed9a9c1ca108f2883959e
- Here: https://reznik.fandom.com/wiki/WotLK_Boost#Download

play well :)
